### PR TITLE
feat: list new gauges and xai-weth farm

### PR DESCRIPTION
**Pull request by ChefBingbong**
---

### Focus of the PR:
This PR focuses on adding new ERC20 tokens and gauge configurations for various chains, including Arbitrum and Polygon ZkEVM.

### Detailed summary:
- Added a new ERC20 token `XAI` with its contract address and details.
- Added a new ERC20 token `INSP` with its contract address and details.
- Updated the import statement for `SerializedWrappedToken` in `types.ts`.
- Added new gauge configurations for various token pairs on Arbitrum and Polygon ZkEVM chains.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by ChefJerry — DISMISSED** [review]

File: packages/farms/constants/arb.ts [hunk 1]
```diff
@@ -220,4 +220,11 @@ export const farmsV3 = defineFarmV3Configs([
     token1: arbitrumTokens.swETH,
     feeAmount: FeeAmount.LOWEST,
   },
+  {
+    pid: 32,
+    lpAddress: '0xf0B860d338E8B5199606322653B83A166d96E417',
+    token0: arbitrumTokens.xai,
+    token1: arbitrumTokens.weth,
+    feeAmount: FeeAmount.MEDIUM,
+  },
```
Comment (chefjackson) [inline packages/farms/constants/arb.ts:229]: put it on top

File: packages/gauges/src/constants/config/prod.ts [hunk 2]
```diff
@@ -2149,4 +2149,64 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    feeTier: FeeAmount.LOW,
+  },
+  {
+    gid: 216,
+    pairName: 'WETH-USDC',
+    address: '0x4A080D9488cE2C8258185d78852275D6d3c2820c',
+    chainId: ChainId.POLYGON_ZKEVM,
+    type: GaugeType.V3,
+    token0Address: polygonZkEvmTokens.weth.address,
+    token1Address: polygonZkEvmTokens.usdc.address,
```
Comment (chefjackson) [inline packages/gauges/src/constants/config/prod.ts:2169]: usdt?

File: packages/gauges/src/constants/config/prod.ts [hunk 3]
```diff
@@ -2149,4 +2149,64 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    address: '0xD43b9dCbB61e6ccFbCFef9f21e1BB5064F1CB33f',
+    chainId: ChainId.POLYGON_ZKEVM,
+    type: GaugeType.V3,
+    token0Address: polygonZkEvmTokens.weth.address,
+    token1Address: polygonZkEvmTokens.usdc.address,
+    feeTier: FeeAmount.LOW,
+  },
+  {
+    gid: 216,
+    pairName: 'WETH-USDC',
```
Comment (chefjackson) [inline packages/gauges/src/constants/config/prod.ts:2164]: usdt

File: packages/gauges/src/constants/config/prod.ts [hunk 4]
```diff
@@ -2149,4 +2149,64 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    token1Address: polygonZkEvmTokens.matic.address,
+    feeTier: FeeAmount.MEDIUM,
+  },
+  {
+    gid: 219,
+    pairName: 'GRAI-USDC',
+    address: '0x39aCc7cf02af19A1eB0e3628bA0F5C48f44beBF3',
+    chainId: ChainId.POLYGON_ZKEVM,
+    type: GaugeType.V3,
+    token0Address: polygonZkEvmTokens.grai.address,
```
Comment (chefjackson) [inline packages/gauges/src/constants/config/prod.ts:2198]: token0 usdc

File: packages/gauges/src/constants/config/prod.ts [hunk 5]
```diff
@@ -2149,4 +2149,64 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    token1Address: polygonZkEvmTokens.usdc.address,
+    feeTier: FeeAmount.LOW,
+  },
+  {
+    gid: 216,
+    pairName: 'WETH-USDT',
+    address: '0x4A080D9488cE2C8258185d78852275D6d3c2820c',
+    chainId: ChainId.POLYGON_ZKEVM,
+    type: GaugeType.V3,
+    token0Address: polygonZkEvmTokens.weth.address,
```
Comment (chefjackson) [inline packages/gauges/src/constants/config/prod.ts:2168]: token0 usdt

Comment (chefjackson) [reply packages/gauges/src/constants/config/prod.ts:2168]: https://zkevm.polygonscan.com/address/0x4A080D9488cE2C8258185d78852275D6d3c2820c#readContract

File: packages/tokens/src/56.ts [hunk 6]
```diff
@@ -2884,4 +2884,12 @@ export const bscTokens = {
     'mCake Token',
     'https://www.pancake.magpiexyz.io/stake',
   ),
+  insp: new ERC20Token(
+    ChainId.BSC,
+    '0x8D279274789CceC8af94a430A5996eAaCE9609A9',
+    18,
+    'INSP',
+    'Inspect Token',
```
Comment (chefjackson) [inline packages/tokens/src/56.ts:2892]:
```suggestion
    'INSPECT',
```

**Review by chefjackson — APPROVED** [review]

**Pull request merged**